sdk/lib/gcc-compat/StringCchCopyNA.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * PROJECT:     GCC c++ support library
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     StringCchCopyNA implementation
+ * COPYRIGHT:   Copyright 202024 Timo Kreuzer <timo.kreuzer@reactos.org>
```

Comment:
Fix the copyright date here as follows:
`COPYRIGHT:   Copyright 2024 Timo Kreuzer <timo.kreuzer@reactos.org>`